xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -86,7 +86,9 @@
      * @return the Mime Body Part object that was added. Returning it allows script to make modifications to that body
      *         part after it's been set (get/set some headers, etc)
      * @param mimeType the mime type of the content parameter
-     * @param content the content to include in the mail, also directly accepts BodyPart objects
+     * @param content the content to include in the mail. The type depends on the mimetype. For example for a mime
+     *        type of {@code text/plain} or {@code text/html}, the content should be a String, for a mime type of
+     *        {@code xwiki/template} it should be a {@link org.xwiki.model.reference.DocumentReference}, etc.
```

Comment:
hmm that's weird that what I added appears as added by you... It's also missing your part about supporting BodyPart independently of the mimetype.

Reply:
Must be because I added it manually to my local fork. I’ll take care of it in a couple hours or so.. thanks!